tests/input_files/elch_1D_electrodes_galvanostat.dat

```diff
@@ -67,8 +67,8 @@ SCATRA DIS scatra NODE 200 QUANTITY phi1 VALUE 1.17933135469771067e+00 TOLERANCE
 SCATRA DIS scatra NODE 200 QUANTITY phi2 VALUE -1.71411948005193533e-03 TOLERANCE 1.8e-11
 -------------------------ELECTRODE DOMAIN KINETICS SURF CONDITIONS
 DSURF  2
-E 1 - ConditionID 0 POT -0.01 FUNCT none NUMSCAL 1 STOICH -1  E- 1 ZERO_CUR 0 Butler-Volmer A_S 1717.5 ALPHA_A 0.5 ALPHA_C 0.5 I0 36.0 GAMMA 0.0 REFCON 1.5 DL_SPEC_CAP 0.0 END
```

Comment:
@c-p-schmidt I took the liberty to remove the useless `END` instead of adding an unnecessary `tag()` for that.